2021-09-20 23:03:05,024 INFO    Thread-8  :550033 [wandb_setup.py:_flush():69] setting env: {'project': 'simple-classification-sweep', 'entity': 'sang-hyun', 'root_dir': '/tf/CI/MyTorchTemplate', 'run_id': 'phzt6sk0', 'sweep_param_path': '/tf/CI/MyTorchTemplate/wandb/sweep-bslbq74a/config-phzt6sk0.yaml', 'sweep_id': 'bslbq74a'}
2021-09-20 23:03:05,025 INFO    Thread-8  :550033 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-20 23:03:05,025 INFO    Thread-8  :550033 [wandb_init.py:_log_setup():342] Logging user logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_230305-phzt6sk0/logs/debug.log
2021-09-20 23:03:05,025 INFO    Thread-8  :550033 [wandb_init.py:_log_setup():343] Logging internal logs to /tf/CI/MyTorchTemplate/wandb/run-20210920_230305-phzt6sk0/logs/debug-internal.log
2021-09-20 23:03:05,025 INFO    Thread-8  :550033 [wandb_init.py:init():375] calling init triggers
2021-09-20 23:03:05,026 INFO    Thread-8  :550033 [wandb_init.py:init():382] wandb.init called with sweep_config: {'NOW': '2021-09-20T23:02:56', 'SAVEFIG_NAME': 'result.png', 'batchsize': 64, 'epochs': 10, 'learning_rate': 0.05760312348262441}
config: {}
2021-09-20 23:03:05,026 INFO    Thread-8  :550033 [wandb_init.py:init():424] starting backend
2021-09-20 23:03:05,026 INFO    Thread-8  :550033 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-20 23:03:05,030 INFO    Thread-8  :550033 [backend.py:ensure_launched():135] starting backend process...
2021-09-20 23:03:05,034 INFO    Thread-8  :550033 [backend.py:ensure_launched():140] started backend process with pid: 550215
2021-09-20 23:03:05,034 INFO    Thread-8  :550033 [wandb_init.py:init():429] backend started and connected
2021-09-20 23:03:05,035 INFO    Thread-8  :550033 [wandb_run.py:_config_callback():843] config_cb None None {'NOW': '2021-09-20T23:02:56', 'SAVEFIG_NAME': 'result.png', 'batchsize': 64, 'epochs': 10, 'learning_rate': 0.05760312348262441}
2021-09-20 23:03:05,035 INFO    Thread-8  :550033 [wandb_init.py:init():477] updated telemetry
2021-09-20 23:03:05,035 INFO    Thread-8  :550033 [wandb_init.py:init():500] communicating current version
2021-09-20 23:03:05,512 INFO    Thread-8  :550033 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-20 23:03:05,512 INFO    Thread-8  :550033 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-20 23:03:05,791 INFO    Thread-8  :550033 [wandb_init.py:init():540] starting run threads in backend
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_run.py:_console_start():1601] atexit reg
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-20 23:03:06,629 ERROR   Thread-8  :550033 [wandb_run.py:_redirect():1539] Failed to redirect.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/wandb_run.py", line 1532, in _redirect
    out_redir.install()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/usr/lib/python3.6/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_init.py:init():565] run started, returning control to user process
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_config.py:__setitem__():141] config set name = sweep-2021-09-20T23:03:06 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7ff8ce3ab908>>
2021-09-20 23:03:06,629 INFO    Thread-8  :550033 [wandb_run.py:_config_callback():843] config_cb name sweep-2021-09-20T23:03:06 None
2021-09-20 23:03:06,630 INFO    Thread-8  :550033 [wandb_run.py:finish():1271] finishing run sang-hyun/simple-classification-sweep/phzt6sk0
